Intersect.Server.Core/Entities/Entity.cs

```diff
@@ -2099,6 +2103,10 @@ public virtual void TryAttack(Entity target)
                 }
 
                 var manastealRate = (thisPlayer.GetEquipmentBonusEffect(ItemEffect.Manasteal) / 100f);
+                   if (manastealRate < 0)
```

Comment:
remove this if, regardless of the value being negative, it will not pass the next check on the line 2113

Reply:
Same situation here. After all, it's just an exception that checks whether the lifestealrate or manastealrate is minus. if so, change it to 0, i.e. the default value. So it should not interfere with the next check regarding +0msg